deploy/packaging/tr1d1um_spruce.yaml

```diff
@@ -123,6 +123,11 @@ log:
 # information using argus.
 webhook:
 
+  # requirePartnerIDs, if true, will require webhooks to have partnerIDs in the request
+  # for them to be registered.
+  # Defaults to 'true'.
+  requirePartnerIDs: true
```

Comment:
A boolean can't default to `true` - its default is always `false` (a boolean can only be two values so the value that is the empty/unset value must be the default).  We need to change the configuration so that requiring partner IDs is the `false` value.

Reply:
A good reference for using zero values is Dave Cheney's blog.  Here's a deep link:
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_make_the_zero_value_useful

It's best to use that principle not only in `golang` code proper but also in configuration.  For example, you can flip the name to `disablePartnerIDs`, which will naturally default to `false`.